tron/api/async_resource.py

```diff
@@ -0,0 +1,48 @@
+import threading
+
+from twisted.internet import threads
+from twisted.web import server
+
+
+class AsyncResource():
+    capacity = 10
```

Comment:
this seems kind of...low. Is this based on you testing?

Reply:
just a wild guess. we can always bump this number if things are too slow and concurrency is the answer.

Reply:
https://twisted.readthedocs.io/en/twisted-18.4.0/core/howto/threading.html suggests that the max thread pool is 10. Maybe that's why larger numbers caused errors?